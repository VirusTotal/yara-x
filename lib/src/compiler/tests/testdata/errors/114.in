rule test {
  strings: 
    $a = "foo" nocase ascii base64wide
  condition:
    $a
}