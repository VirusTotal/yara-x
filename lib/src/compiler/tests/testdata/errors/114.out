error: invalid modifier combination: `base64wide` `nocase`
 --> line:3:29
  |
3 |     $a = "foo" nocase ascii base64wide
  |                             ^^^^^^^^^^ `base64wide` modifier used here
  |                ^^^^^^ `nocase` modifier used here
  |
  = note: these two modifiers can't be used together